Background-Substractor-Video-Capturer/README.md

```diff
@@ -77,14 +77,14 @@ Since it's good practice to create a build folder, let's go ahead and create it
 in the project directory:
 
 ```bash
-$ mkdir Background-Substractor-Video-Capturer/build
+mkdir Background-Substractor-Video-Capturer/build
```

Comment:
Removing the `$`, to be consistent with the rest of the README.

Reply:
We have a mixture of using them and not using them across all README files in the repo. Not a big deal IMO.